tests/phpunit/api/v3/CustomApiTest.php

```diff
@@ -60,8 +60,8 @@ public function testCustomApi() {
     $this->callAPISuccess('Mailing', 'create', array('name' => 'CiviMail', 'hash' => 'abx'));
     $result = $this->callAPISuccess('MailingProviderData', 'get', array('return' => array('mailing_identifier.name', 'contact_identifier', 'mailing_identifier')));
     $this->assertEquals(1, $result['count']);
-    $this->assertEquals('xyzabx0000-00-00 00:00:00', $result['id']);
-    $this->assertEquals('xyzabx0000-00-00 00:00:00', $result['id']);
+    $this->assertEquals('xyzabx2017-01-01 00:00:00', $result['id']);
+    $this->assertEquals('xyzabx2017-01-01 00:00:00', $result['id']);
```

Comment:
this is about no zero date specifically so just set it to the first of Jan 2017. Test then passes locally